integrations/adobe-analytics/lib/heartbeat.js

```diff
@@ -0,0 +1,180 @@
+
+var heartbeats = {
```

Comment:
Thanks for breaking this out! I think this is specific to chromecast heartbeat which makes sense. I think we can make that more explicit in the name of the object and filename.

```suggestion
var chromecastHeartbeatUtil = {
```